ops-bedrock/entrypoint-l1.sh

```diff
@@ -14,9 +14,9 @@ WS_PORT="${WS_PORT:-8546}"
 
 if [ ! -d "$GETH_KEYSTORE_DIR" ]; then
 	echo "$GETH_KEYSTORE_DIR missing, running account import"
-	echo -n "pwd" > "$GETH_DATA_DIR"/password
-	echo -n "$BLOCK_SIGNER_PRIVATE_KEY" | sed 's/0x//' > "$GETH_DATA_DIR"/block-signer-key
-	geth account import \
+	echo -n "%s" "pwd" > "$GETH_DATA_DIR"/password
```

Comment:
It was when the shebang was `#!/bin/sh`, but we changed that to `#!/bin/bash` in ba7d2092e per https://github.com/ethereum-optimism/optimism/pull/8594#discussion_r1424795445, and shellcheck no longer flags it since `-n` is supported in bash. 

However, that change caused CI to fail, which I have not looked into yet

Reply:
Resolving since the `"%s"` part of that command has been deleted in a6462d4, and CI is now passing